modules/payments/client-react/stripe/subscription/containers/UpdateCreditCard.tsx

```diff
@@ -36,29 +36,23 @@ class UpdateCreditCard extends React.Component<UpdateCreditCardProps, { [key: st
 
     try {
       // create credit card token
-      try {
-        preparedCreditCard = await createCreditCardToken(creditCardInput, stripe);
-        if (preparedCreditCard.error) {
-          this.setState({ submitting: false, error: t('stripeError') });
```

Comment:
Looks like you forgot to remove "error" field from the component state and from the render function

Reply:
Fixed